packages/phasor/src/elements/shape/shapes/diamond.ts

```diff
@@ -9,26 +9,31 @@ import type { ShapeMethods } from '../types.js';
 export const DiamondMethods: ShapeMethods = {
   render(
     ctx: CanvasRenderingContext2D,
+    matrix: DOMMatrix,
```

Comment:
@Flrande are there `render` methods still being used after you switch to roughjs?